public/api/conf/1.0/application.yaml

```diff
@@ -656,22 +745,38 @@ paths:
                     name: CustomsDeclarationsArrivalNotificationresponse400
                     attribute: false
                     wrapped: false
-        '500':
+        '403':
           description: ''
           headers: {}
           content:
             application/xml:
               examples:
                 objectExample:
-                  $ref: '#/components/examples/InternalServerErrorExample'
+                  $ref: '#/components/examples/PayloadForbiddenExample'
               schema:
                 allOf:
                 - $ref: '#/components/schemas/CustomsDeclarationsArrivalNotificationresponse400'
-                - description: Internal server error
+                - description: A firewall rejected the request
                   xml:
                     name: CustomsDeclarationsArrivalNotificationresponse400
                     attribute: false
                     wrapped: false
+        '500':
+          description: ''
+          headers: {}
+          content:
+            application/xml:
+              examples:
+                objectExample:
+                  $ref: '#/components/examples/InternalServerErrorExample'
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/CustomsDeclarationsArrivalNotificationresponse400'

```

Comment:
Indentation inconsistent with rest of file.

Reply:
indentation is the same ad the 400 errors

Reply:
run it through here and if it passes you're good https://editor.swagger.io

Reply:
done